spec/timezone.html

```diff
@@ -191,6 +191,11 @@ <h1>Abstract operations</h1>
       <h1>TimeZoneFromString ( _timeZone_ )</h1>
       <emu-alg>
         1. Assert: Type(_timeZone_) is String.
+        1. If _timeZone_ matches `/([+-][01]?[0-9]):?([0-5][0-9])/`, then
```

Comment:
What is the rationale for these bounds? -19:59 through +19:59 is a strange cutoff.

Reply:
I believe this came from current maximum being +14:00

Such that I didn’t add a third option for the first numeric digit in parsing.

The absolute maximum would be +-24:00 is the theoretical possibility of that more important?

Reply:
This line I've just copied from another place in the spec (the Temporal.TimeZone constructor) assuming it'll get replaced as part of #312.

Reply:
ISO 8601 does not establish bounds as far as I can tell, so I'd align with [RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6) on -23:59 through +23:59.

Reply:
OK, will address this as part of #312 then.